code/__HELPERS/game.dm

```diff
@@ -14,18 +16,21 @@
 
 /// Checks all conditions if a spot is valid for construction , will return TRUE
 /proc/is_valid_for_resin_structure(turf/target, needs_support = FALSE, mob/builder)
-
 	if(!target || !istype(target))
 		return ERROR_JUST_NO
 	var/obj/alien/weeds/alien_weeds = locate() in target
 	if(!target.check_disallow_alien_fortification(null, TRUE))
 		return ERROR_NOT_ALLOWED
 	if(!alien_weeds)
 		return ERROR_NO_WEED
+	if(get_build_prereqs(target))
+		return ERROR_INVALID_AREA
 	if(!target.is_weedable())
 		return ERROR_CANT_WEED
 	for(var/obj/effect/forcefield/fog/F in range(1, target))
 		return ERROR_FOG
+	if(!length(GLOB.xeno_resin_silos_by_hive[XENO_HIVE_NORMAL]))
+		return ERROR_NO_SILO
```

Comment:
Doesn't this mean you simply can't build at all until a silo is dropped? i.e. you can no longer wait to drop a silo until after you see what fob the marines are using etc?

Reply:
Yes, but it's a simple fix. I can do that immediately if needed.